### PR TITLE
silence make-disk-image tests

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

**Review by Lassulus — COMMENTED** [review]
We cant drop that yet or we will crash on 23.05

File: lib/tests.nix [hunk 1]
```diff
@@ -173,7 +173,7 @@ let
             })
           ];
 
-          boot.initrd.services.swraid.enable = true;
+          boot.swraid.enable = true;
```
Comment (Lassulus) [inline lib/tests.nix:176]: This also needs to be wrapped like in mdadm.nix. since we export the rest function in the module

Comment (Mic92) [reply lib/tests.nix]: Done.

**Comment by Lassulus** [comment]
@mergify queue

**Pull request merged**